modules/service/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -4262,16 +4244,10 @@ type Observation {
   constraintSet: ConstraintSet!
 
   # The top level science requirements
-  scienceRequirements(
-    # Set to true to include deleted values
-    includeDeleted: Boolean! = false
-  ): ScienceRequirements!
```

Comment:
This and the old `scienceMode` below had a spurious `includeDeleted` flag that wasn't used even in the tmp-api.